packages/authorization-process/src/services/readModelService.ts

```diff
@@ -400,6 +410,33 @@ export function readModelServiceBuilder(
       }
       return undefined;
     },
+
+    async getActiveConsumerDelegationById(
+      id: DelegationId
+    ): Promise<Delegation | undefined> {
+      const data = await delegations.findOne(
+        {
+          "data.id": { $eq: id },
+          "data.state": { $eq: delegationState.active },
+          "data.kind": { $eq: delegationKind.delegatedConsumer },
```

Comment:
Should work also without `$eq`?

```suggestion
          "data.id": id,
          "data.state":  delegationState.active,
          "data.kind": delegationKind.delegatedConsumer,
```